src/pages/index.astro

```diff
@@ -158,4 +158,16 @@ const examples = (await getCollection('example'));
       </Card>
     </div>
   </Section>
+
+  <!-- CNCF & LF -->
+  <Section class="bg-base-300">
+    <h2 class="text-primary text-3xl font-bold text-center mb-4">Proudly Open Source</h2>
```

Comment:
Prouldy Open Source sounds a bit cliché to me. I'd rather go with:

```suggestion
    <h2 class="text-primary text-3xl font-bold text-center mb-4">Innovation in the Open</h2>
```

Reply:
Done